test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorPageTest.cs

```diff
@@ -1188,7 +1188,7 @@ public static TheoryData WriteAttributeData
             return view.Object;
         }
 
-        private static ViewContext CreateViewContext(TextWriter writer = null)
+        private static ViewContext CreateViewContext(TextWriter writer = null, string viewPath = null)
```

Comment:
Ignore previous comment here since product code _will_ be using `IView.Path` in some tests.